cluster-up/hack/common.sh

```diff
@@ -26,7 +26,7 @@ KUBEVIRT_PSA=${KUBEVIRT_PSA:-false}
 KUBEVIRT_SINGLE_STACK=${KUBEVIRT_SINGLE_STACK:-false}
 KUBEVIRT_ENABLE_AUDIT=${KUBEVIRT_ENABLE_AUDIT:-false}
 KUBEVIRT_DEPLOY_NFS_CSI=${KUBEVIRT_DEPLOY_NFS_CSI:-false}
-KUBEVIRT_DEPLOY_PROMETHEUS=${KUBEVIRT_DEPLOY_PROMETHEUS:-false}
+KUBEVIRT_DEPLOY_PROMETHEUS=${KUBEVIRT_DEPLOY_PROMETHEUS:-true}
```

Comment:
I don't think we should change this default

Reply:
If we want the alerts to be deployed by default, why not?

Reply:
This is in the local dev environment, most devs don't use the monitoring stack for daily tasks, so no need to further load their systems. If they want setting the env themselves before creating the cluster it's a simple step

Reply:
Removed it.